modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -0,0 +1,219 @@
+= Guide to Named Examples
+
+If you want to use more than one example for a type, resource type, or trait, you must name the examples. You can also name a single example. When you name examples, you must use the `examples` facet.
+
+To use one example without naming it, you must use the `example` facet.
+
```

Comment:
I think the previous four sentences sort of waver back and forth between naming more than one, naming one, how to name, and how to not name.
How about "You must use the examples facet to name examples, whether you have one or many. You must also use the example facet to use one example without naming it."
How odd!

Reply:
@kbostrom , I've rewritten this intro.